index.js

```diff
@@ -37,15 +37,15 @@ async function checkPair(args) {
   const exchangeContract = new web3.eth.Contract(UNISWAP_EXCHANGE_ABI, exchangeAddress)
 
   const uniswapResult = await exchangeContract.methods.getEthToTokenInputPrice(inputAmount).call()
-  let kyberResult = await kyberRateContract.methods.getExpectedRate(inputTokenAddress, outputTokenAddress, inputAmount, true).call()
+  let kyberResult = await kyberRateContract.methods.getExpectedRate(inputTokenAddress, outputTokenAddress, inputAmount).call()
 
   console.table([{
     'Input Token': inputTokenSymbol,
     'Output Token': outputTokenSymbol,
     'Input Amount': web3.utils.fromWei(inputAmount, 'Ether'),
     'Uniswap Return': web3.utils.fromWei(uniswapResult, 'Ether'),
     'Kyber Expected Rate': web3.utils.fromWei(kyberResult.expectedRate, 'Ether'),
-    'Kyber Min Return': web3.utils.fromWei(kyberResult.slippageRate, 'Ether'),
+    'Kyber Min Return': web3.utils.fromWei(kyberResult.worstRate, 'Ether'),
```

Comment:
Does mean that it will just pick the worst price instead of slippageRate as before? Is there a better way of doing this?

Reply:
oh wow that is very interesting, i didn't notice this...
do we have answer to your question yet?